src/raw.rs

```diff
@@ -232,18 +232,59 @@ pub fn string_dma(key: *mut RedisModuleKey, len: *mut size_t, mode: KeyMode) ->
     unsafe { RedisModule_StringDMA.unwrap()(key, len, mode.bits) }
 }
 
-pub fn hash_get(key: *mut RedisModuleKey, field: &str) -> *mut RedisModuleString {
-    let res: *mut RedisModuleString = ptr::null_mut();
-    unsafe {
-        RedisModule_HashGet.unwrap()(
-            key,
-            REDISMODULE_HASH_CFIELDS as i32,
-            CString::new(field).unwrap().as_ptr(),
-            &res,
-            ptr::null::<c_char>(),
-        );
+pub fn hash_get_multi<T>(key: *mut RedisModuleKey, fields: &[T],
+                         values: &mut [*mut RedisModuleString])
+    -> Result<(), RedisError>
+    where T: Into<Vec<u8>> + Clone
+{
+    assert_eq!(fields.len(), values.len());
+
+    let mut fi = fields.iter();
+    let mut vi = values.iter_mut();
+
+    macro_rules! RM {
```

Comment:
`RM` is not a valid name for a macro; should be lowercase/snake_case instead.
Are you using the IntelliJ Rust plugin? Or VS Code? These should report things like this.